.github/workflows/release.yaml

```diff
@@ -0,0 +1,30 @@
+name: bincapz GoReleaser
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
make this: `- 'v*'`

Reply:
Addressed in [`4899d0c` (#125)](https://github.com/chainguard-dev/bincapz/pull/12!5/commits/4899d0c62ce1df7bc4595f7255432820ffe55c7a)!